www/style.css

```diff
@@ -35,6 +35,21 @@ sub {
   display: inline-block;
 }
 
+#output-text {
+  width: 500px;
+  height: 175px;
+}
+
+#code-output {
+  width: 500px;
+  height: 175px;
+}
+
+#target-text {
+  width: 500px;
+  height: 175px;
+}
+
```

Comment:
Should this be a unified style for every `textarea` rather than 3 separate rules?